contracted-contributor.md

```diff
@@ -0,0 +1,161 @@
+## Title
+
+_Contracted Contributor_ (aka _Free Electron_)
+
+## Problem
+
+Company A has started an InnerSource initiative. One of the goals for this
+initiative is to foster innovation by allowing every associate to voluntarily
+contribute to InnerSource projects, regardless of topic and business unit. 
+After a year, only a few voluntary contributions have been received. While
+engaging with potential individual contributors, the [[Dedicated Community Leader]]
+determined that the main problem was that the contributors respective line 
+managers did not allow them to contribute.
+
+As a result, the total number of contributors remained small and progress was
+not as fast as expected by top level management. Furthermore, contributions
+mostly originated in the same business unit the Dedicated Community Leader
+belonged to.  Innovation did not happen in the expected scale. Top management
+is no longer convinced that InnerSource yields the expected benefits and
+abandons the initiative altogether.
+
+**Review Comments**
+
+* (done) Clarify: "Not allowed to contribute"--by line managers (done)
+* (done) **Introduce what was observed; then note that the main problem is XYZ**
+* (done with previous two points) Seems to be several problems: not getting many contributions, line manager issue, top management not convinced that they are getting the benefits (how do you keep top management happy). Think about: what is really different; what part of the problem statement is solved by the solution.
+
+## Context
+
+- Top level management was on board with and is supporting the InnerSource initiative. For them, InnerSource initiative is just one of many initiatives to foster innovation. They are funding InnerSource with money and capacity for community leaders and are giving autonomy as for how the budget is spent. They are also limiting the breadth and duration of the initiative and partake in periodic reviews until there is proof that it yields the expected results. Top level management has announced their support for InnerSource on various company internal meetings, but has not empowered or incentivised mid-level managers to allow their employees to participate in cross-divisional InnerSource activities.
+- The capacity of every associate is usually allocated to non InnerSource
+  projects for 100 %.
+- Cross organizational collaboration is not the norm.
+- Line managers usually do not have targets outside of their own organisation.
+- Contributions to InnerSource projects are expected to be made during working
+  hours, not during free time.
+
+**Review comments**
+
+* **(done)** Top level management is aware, expecting savings, putting in money. Is this budget being put into saving capacity for InnerSource? Is a barrier preventing this? Where is that money going? People don't have to hide from top level management.
```

Comment:
I didn't notice these in there. I just wholesale copied over for the time being.

We discussed in the meeting that we would like to have ongoing patterns included into master sooner. You look at the repo and it looks like nothing is going on if you dont know to dig into the ongoing PR's. I think we should initially accept pattern PR's, have something out there, and then work on modifications and comments to that base in additional PR's in the same way.